compiler+runtime/test/jank/cpp/try/pass-catch-cpp-std-out-of-range.jank

```diff
@@ -0,0 +1,13 @@
+;; Test catching std::out_of_range
```

Comment:
This test is redundant with the `bad_cast`, `invalid_argument`, and `logic_error` tests. We don't need all four.